### PR TITLE
New Homepage Design

### PR DESCRIPTION
**Pull request by jvsluis**
Added new homepage. Colours will need to be updated once we decide on the final colour scheme.

image

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -27,6 +29,91 @@ function Layout() {
...
+        </nav>
+      );
+    }
+
+    function standardNav() {
+      return (
+        <nav className="flex justify-between items-center w-full px-4 py-3 app-page">
+          <Link to="/">
+            <h2 className="text-4xl">
+              <span className="fg-brand-blue">Print</span><span className="fg-brand-purple">Link3D</span>
```
Comment (Jasper-Nelligan) [inline src/app.js]: Should be PrintLink3d

File: src/components/homePageStep.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
-.step-circle {
```
Comment (Jasper-Nelligan) [inline src/components/homePageStep.css:1]: Is this file necessary anymore since all the code was put in app.css?

File: src/routes/home.js [hunk 3]
```diff
@@ -1,95 +1,81 @@
...
-              className="w-1/2 sm:w-full mx-auto"
-            />
+          <Link to="/create" className="mt-8 bg-brand-purple text-white p-4 px-6 rounded">Get Started</Link>
+        </div>
+        <div id="ihaveprinter">
+          <div className="flex items-center p-12 rounded" style={{backgroundColor: '#F7F7F7'}}>
+            <div className="w-[40%] pr-12">
+              <h2 className="font-bold text-5xl mb-8">I Have a 3D Printer</h2>
+              <p className="text-xl mb-8">Select a job in your local area and start making extra money! It’s as easy as that.
+                 <br/><br/>Our mission at PrintLink3D is to connect customers with local 3D printing enthusiasts. Interested? </p>
```
Comment (Jasper-Nelligan) [inline src/routes/home.js]: PrintLink3d

File: src/routes/home.js [hunk 4]
```diff
@@ -1,95 +1,81 @@
...
-              printing enthusiasts. Interested?
-            </p>
-            <div className="flex justify-center">
-              <Link to="/browse" className="mt-10 bg-custom-purple hover:bg-custom-purple-hover border-black text-white font-bold py-3 px-4 rounded-lg text-2xl sm:text-1xl md:text-2xl lg:text-3xl xl:text-4xl 2xl:text-5xl">
-                Get Started
-              </Link>
+      <div className="container m-auto">
+        <div id="iwantprint" className="flex flex-col items-center justify-center py-[100px]">
+          <h2 className="font-bold text-5xl pb-8">Getting Your Model Printed</h2>
+          <div className="flex flex-row gap-12 w-full py-8">
```
Comment (Jasper-Nelligan) [inline src/routes/home.js]: For responsiveness it'd be nice to have the steps be vertically placed on small screens.

Overall this is a major improvement over what we had before!

Comment (jvsluis) [reply src/routes/home.js]: I fixed the mobile view for the homepage and its nav bar. Starting to look better now! :)

**Review by Jasper-Nelligan — APPROVED** [review]
